app/blog/view-counter.tsx

```diff
@@ -24,7 +24,7 @@ export default function ViewCounter({
   trackView: boolean;
 }) {
   const { data } = useSWR<PostView[]>('/api/views', fetcher);
-  const viewsForSlug = data && data.find((view) => view.slug === slug);
+  const viewsForSlug = data?.find?.((view) => view.slug === slug);
```

Comment:
Isn't this doing the same thing?

Reply:
@leerob it's not, just after executing `pnpm run setup`, `data` would be an object (not an array) like so: `{ message: 'fetch failed' }`, so the extra chaining for `find?.()` will fix the crash of `/blog` page.